main.bicep

```diff
@@ -32,3 +31,68 @@ resource websiteconfig 'Microsoft.Web/staticSites/config@2021-02-01' ={
     StorageConnectionString: 'DefaultEndpointsProtocol=https;AccountName=${storage.name};AccountKey=${storage.listKeys().keys[0].value};EndpointSuffix=${environment().suffixes.storage}'
   }
 }
+
+resource storageBlobServiceConfig 'Microsoft.Storage/storageAccounts/blobServices@2021-08-01' = {
+  parent: storage
+  name: 'default'
+  properties: {
+    lastAccessTimeTrackingPolicy:{
+      enable: true
+    }
+    cors: {
+      corsRules: [
+        {
+          allowedOrigins: [
+            website.properties.defaultHostname
```

Comment:
Enable staging slots too

Reply:
this isn't possible right now. Or not really.
The following ist the correct wildcard schema that represents azure static websites stages:

```bicep
'${split(website.properties.defaultHostname, '.')[0]}-*.${location}.azurestaticapps.net'
```

The problem is this is not allowed by the cors rules of an storage account.
The only solution is to deploy it very unsafe in a the following way:

```bicep
'*.${location}.azurestaticapps.net'
```

or to add every stage hostname to the cors rules after deploying the stage to azure. But this would require the following steps:

1. provisioning
   1. get all cors rules from the storage account
   1. add the one for the current stage to the config
   1. deploy it again
1. deprovisioning
   1. get all cors rules from the storage account
   1. remove the one of the current Pullrequest
   1. deploy it again

Reply:
I will now use `'*.${location}.azurestaticapps.net'` as a workaround to close this for now...

Reply:
I will now use '*.${location}.azurestaticapps.net' as a workaround to close this for now...